stomp_moveit/include/stomp_moveit/cost_functions/stomp_cost_function.h

```diff
@@ -109,7 +109,7 @@ class StompCostFunction
                             int iteration_number,
                             int rollout_number,
                             Eigen::VectorXd& costs,
-                            bool& validity) = 0 ;
+                            bool& validity) const = 0 ;
```

Comment:
Too bad OpenMP requires using const functions, this doesn't allow the use of class members and in this case a couple of public interface methods need to be modified.